olp-cpp-sdk-core/include/olp/core/client/ApiResponse.h

```diff
@@ -28,20 +28,20 @@ namespace olp {
 namespace client {
 
 /**
- * Template class representing the outcome of making a request.  It will contain
- * either a successful result or the failure error.  The caller must check
- * whether the outcome of the request was a success before attempting to access
- *  the result or the error.
+ * Represents a request outcome.
+ *
+ * Contains a successful result or the failure error. Before you try to access
```

Comment:
```suggestion
 * Contains a successful result or failure error. Before you try to access
```
Sorry, I thought there was only one failure error.